Detailed-README.md

```diff
@@ -372,6 +372,40 @@ Note: options passed through cli have higher priority than those specified in th
 ```
 bundle exec bin/splitio -h
 ```
+## Server support
+
+Currently SDK supports:
+  - Thin
+  - Puma
+  - Passenger
+  - Unicorn
+
+Other servers should work fine as well, but haven't been tested.
+
+### Unicorn
+
+If you're using Unicorn in the `memory` mode you'll need to include this line in your unicorn config (probably `config/unicorn.rb`):
+
+```ruby
+after_fork do |server, worker|
+  Rails.configuration.split_factory.run_adapter! if worker.nr > 0
```

Comment:
can we call it `resume` instead?

it, for now, can be an alias to `start` or `initialize` but it gives us something to evolve it in the future.